### ISSUE TITLE
Aegis crashes when adding an icon (for the 17th item)

### ISSUE THREAD

**Issue by elieobeid7** [beemdevelopment/Aegis#648]
###### Info

* __Version__:  1.4 (latest stable version, I'm not a beta user)
* __Source__: Google Play store
* __Vault encrypted__: Yes (with biometric unlock, yes fingerprint enabled)
* __Device__: Google pixel 4a
* __Android version__ 11, my device has the latest official updates, no custom ROM, not rooted

###### Steps to reproduce

I have 16 items with icons, added successfully, I just added a 17th item, I'm trying to add an icon for the 17th item, if I press on google photos or the file manager the app crashes, so I see the popup (select an icon) asking me to either select google photos or file manager if I click on either of those the app crashes immediately.

Again everything is normal for the 16 previous items, I started having this issue when adding the 17th item.

**Comment by alexbakker** [comment]
Could you capture a log of the crash with ADB and share it here? We have a guide on how to do so here: https://github.com/beemdevelopment/Aegis/blob/master/CONTRIBUTING.md.

**Comment by elieobeid7** [comment]
@alexbakker  I get this output when pressing on the file manager and google photos

```
--------- beginning of main
01-09 22:03:35.581 17573 17573 E velopment.aegi: Not starting debugger since process cannot load the jdwp agent.
01-09 22:03:35.603 17573 17573 D NetworkSecurityConfig: No Network Security Config specified, using platform default
01-09 22:03:35.603 17573 17573 D NetworkSecurityConfig: No Network Security Config specified, using platform default
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: QUALCOMM build                   : 0905e9f, Ia11ce2d146
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: Build Date                       : 09/02/20
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: OpenGL ES Shader Compiler Version: EV031.31.04.00
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: Local Branch                     : gfx-adreno.lnx.2.0
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: Remote Branch                    : 
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: Remote Branch                    : 
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: Reconstruct Branch               : 
01-09 22:03:35.789 17573 17594 I AdrenoGLES-0: Build Config                     : S P 10.0.4 AArch64
01-09 22:03:35.790 17573 17594 I AdrenoGLES-0: Driver Path                      : /vendor/lib64/egl/libGLESv2_adreno.so
01-09 22:03:35.800 17573 17594 I AdrenoGLES-0: PFP: 0x016ee189, ME: 0x00000000
01-09 22:03:35.804 17573 17594 W AdrenoUtils: <ReadGpuID_from_sysfs:197>: Failed to open /sys/class/kgsl/kgsl-3d0/gpu_model
01-09 22:03:35.804 17573 17594 W AdrenoUtils: <ReadGpuID:221>: Failed to read chip ID from gpu_model. Fallback to use the GSL path
01-09 22:03:35.816 17573 17594 I Gralloc4: mapper 4.x is not supported
01-09 22:03:46.760 17573 17573 W Choreographer: Frame time is 0.009705 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
01-09 22:03:46.793 17573 17573 W Choreographer: Frame time is 0.042012 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
01-09 22:03:46.826 17573 17573 W Choreographer: Frame time is 0.031773 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
01-09 22:03:55.494 17573 17573 I VaultBackupManager: Shutting down backup manager thread

```

**Comment by alexbakker** [comment]
@elieobeid7 This does not contain the stack trace of a crash.

**Comment by elieobeid7** [comment]
But I did exactly as mentioned, opened the app then

 adb logcat --pid=$(adb shell pidof -s com.beemdevelopment.aegis) > debug.log

then tried to add an icon. Anything else you want me to do?

**Comment by alexbakker** [comment]
Hmm, that's odd. Could you retry without the ``--pid=$(adb shell pidof -s com.beemdevelopment.aegis)`` part?

**Comment by elieobeid7** [comment]
@alexbakker [here's the log](https://pastebin.com/7p6w2R83)

**Comment by alexbakker** [comment]
Still no crash, but I think I know what's going on here. Do you have "The app is minimized" enabled in the auto lock settings? If so, #643 should fix the issue, and you can temporarily work around it by disabling that option until the next version is released

**Comment by elieobeid7** [comment]
@alexbakker that's true, I disabled that option and the issue is fixed, thank you so much! do you prefer that I close this issue now or will you close it when the next version is released?

**Comment by alexbakker** [comment]
It'll be closed automatically after #643 is merged.
